appveyor.yml

```diff
@@ -20,8 +20,22 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
+      CONDA_PY: 27
 
     - TARGET_ARCH: x64
+      CONDA_PY: 27
+
+    - TARGET_ARCH: x86
+      CONDA_PY: 34
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 34
+
+    - TARGET_ARCH: x86
+      CONDA_PY: 35
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 35
```

Comment:
I think there was a bug in the original rendering, as this is correct: it should be built for all arch/runtimes not just all arches ; we did not notice as the feedstock is not used in openturns for windows.